include/model.php

```diff
@@ -214,14 +214,14 @@ public function get_terms_args( $args ) {
 	 *
 	 * @since 1.2
 	 *
-	 * @param string[]           $clauses The list of sql clauses in terms query.
-	 * @param PLL_Language|false $lang    PLL_Language object.
-	 * @return string[]                   Modified list of clauses.
+	 * @param string[]       $clauses   The list of sql clauses in terms query.
+	 * @param PLL_Language[] $languages PLL_Language object.
```

Comment:
This method is used [in PLL for WC](https://github.com/polylang/polylang-wc/blob/f70ed62313ef8473a98bf556ea7e9bf6e3f4acc5/src/admin/admin-reports.php#L189) and injects a `PLL_Language` object.

```suggestion
	 * @param string[]                    $clauses   The list of sql clauses in terms query.
	 * @param PLL_Language[]|PLL_Language $languages `PLL_Language` object or array of `PLL_Language` objects.
```

Also, since it's public, we should add:

```
	 * @since 3.8 Accepts an object or an array of objects.
```